js/main.js

```diff
@@ -25,9 +25,86 @@ function getRandomFractionNumber(min, max, numberSymbols) {
 
   const randomNumber = min + Math.random() * (max - min);
 
-  return Number(randomNumber.toFixed(1 * numberSymbols));
+  return Number(randomNumber.toFixed(numberSymbols));
 }
 
-getRandomFractionNumber(1, 10, 2);
+getRandomFractionNumber(1, 10, 0);
 
 
+// Возвращает фотографию пользователя
+const AVATAR_COUNTS = 10;
+
+function getAvatarUrl () {
+  const randomNumber = getRandomFractionNumber(1, AVATAR_COUNTS, 0);
+
+  let AvatarUrl = 'img/avatars/userget' + `0${ randomNumber }.png`;
+  if (randomNumber === 10) {
```

Comment:
Ну и я предлагаю заменить все на 
```suggestion
  let AvatarUrl = `img/avatars/userget${ String(randomNumber).padStart(2,0) }.png`;
```
Но даже если не менять, нужно тогда напимать `if (randomNumber > 10)`, но лучше вообще отказаться от такого, чем меньше цифр волшебных в коде, таких как 10, тем лучше.

Reply:
String(randomNumber).padStart(2,0) Это ты просто переводишь число в строку и добавляешь 0 слева, если количество символов меньше 2